src/bundle/DependencyInjection/Configuration/Parser/Notifications.php

```diff
@@ -17,12 +17,14 @@
  *
  * Example configuration:
  * ```yaml
- * ezpublish:
+ * ibexa:
  *   system:
  *      admin_group: # configuration per siteaccess or siteaccess group
  *          notifications:
  *              warning: # type of notification
  *                  timeout: 5000 # in milliseconds
+ *          notification_count:
+ *               interval: 60000 # in milliseconds
```

Comment:
I would have prefere the following but couldn't do it, see https://github.com/ibexa/admin-ui/pull/1198/files#r1523327082
```yaml
notifications:
    warning: # type of notification
        timeout: 5000 # in milliseconds
    count:
        interval: 60000 # in milliseconds
```